### PR TITLE
Update mod_proxy_cluster demo, examples, configurations

### LINKED ISSUE #141: Update mod_proxy_cluster demo/examples
It is necessary to 

- [x] check that examples are working
- [x] fix configurations (e.g., do not use mod_cluster_slotmem)
- [x] update steps and descriptions

### PR DESCRIPTION
**Pull request by jajik**
close #141

### CODE REVIEW

**Comment by jajik** [comment]
I think it's in a mergeable state, but once we releaase mod_proxy_cluster 2.0, I believe it would be worth it redoing the documentation a bit (there is duplication of information, we could then drop 1.3.x schism, update examples, ...). But that's for sometime in the future...

File: docs/src/main/asciidoc/native.adoc [hunk 1]
```diff
@@ -88,9 +91,9 @@ Check that the Alias corresponds to the ServerName (See http://labs.jboss.com/fi
 
 *Default:* Off Ignore the Alias information from the nodes.
 
-NOTE: *Older mod_cluster versions:* Versions older than 1.3.1.Final only accept values 0 and 1 respectively.
+NOTE: *Older mod_cluster versions:* Versions older than 1.3.1 only accept values 0 and 1 respectively.
```
Comment (rhusar) [inline docs/src/main/asciidoc/native.adoc:94]: This change is wrong, the release name is indeed 1.3.1.Final

https://github.com/modcluster/mod_cluster/tree/1.3.1.Final?search=1
https://github.com/modcluster/mod_cluster/blob/1.3.1.Final/native/include/mod_proxy_cluster.h#L31
https://github.com/modcluster/mod_cluster/blob/1.3.1.Final/pom.xml#L33

**Review by rhusar — CHANGES_REQUESTED** [review]

**Review by rhusar — APPROVED** [review]
LGTM

**Comment by rhusar** [comment]
Thanks @jajik

**Pull request merged**